pkg/kf/commands/root.go

```diff
@@ -153,3 +155,14 @@ func builtIn() bool {
 	flags.Parse(os.Args)
 	return *result
 }
+
+func completionCommand(rootCmd *cobra.Command) *cobra.Command {
```

Comment:
I think it's common practice to have a sub command for each shell. even though we only have bash, I think we should stick to this practice:

Some examples:
```
eval "$(kubectl completion bash)"
eval "$(helm completion bash)"
eval "$(pip completion --bash)"
eval "$(npm completion bash)"
```

Reply:
Good idea. Cobra supports bash and zsh. I'll add support for zsh.